pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -2350,83 +2350,6 @@ func TestResolvePipelineRun_ResourceBindingsDontExist(t *testing.T) {
 	}
 }
 
-func TestResolvePipelineRun_withExistingTaskRuns(t *testing.T) {
```

Comment:
I don't think this test case can be removed, because we still need to test the interaction between pipelineruns and taskruns even if the full status is no longer copied over. Unless there is something specific to this test that covers logic that's only relevant to populating full/both embedded status?

Reply:
That makes sense, this shall be restored.
Thank you Lee.